lib/broadway_rabbitmq/producer.ex

```diff
@@ -423,7 +426,8 @@ defmodule BroadwayRabbitMQ.Producer do
         end
       catch
         kind, reason ->
-          Logger.error(Exception.format(kind, reason, __STACKTRACE__))
+          formatted_error = Exception.format(kind, reason, __STACKTRACE__)
+          Logger.error("Could not ack/reject message: #{formatted_error}")
```

Comment:
Main change in the PR: this makes it clear that the error was trapped by broadway_rabbitmq